### DISCUSSION TITLE
GitHub Student Developer Pack Verification Issue

### DISCUSSION THREAD

**Discussion by chamals3n4** [community/community#144401] [GitHub Education]
### Hi everyone,

I’m having trouble activating the GitHub Student Developer Pack on my personal account. My university requires us to use our university email to create a separate GitHub account for submitting assignments via GitHub Classroom. Because of this, I can’t add my campus email as a secondary email to my personal account, which is needed to verify the Student Pack. Has anyone else had this issue, or is there a way to resolve it?

Thank you!

**Comment by davevad93** [comment]
Hi @chamals3n4 ,

If your academic email is already being used by another GitHub account you own, the solution to get the `Student Developer Pack` would be to submit the request with that account.

**Reply by chamals3n4** [reply]
Thank you for the response and suggestion, I’ve managed to resolve this issue with my university. They’ve agreed to link my personal account instead of using my campus email for lab submission, soo now i can use campus email to verify my developer pack.

**Reply by davevad93** [reply]
You're welcome @chamals3n4 , glad you found a solution! 👍
